<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageConverter</class>
 <widget class="QMainWindow" name="ImageConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageConverter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Open_File">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
border-radius:30px;
color: rgb(255, 255, 0);
border-style:outset;
border-width:2px;
border-color:black;
</string>
    </property>
    <property name="text">
     <string>Select File
 to be Converted</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Open_File">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>841</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>File to be Converted:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_imageFormat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	background-color: rgb(255, 255, 255);
	
	color: rgb(0, 0, 255);
    margin-right: 20px;
}
QComboBox::drop-down {
    subcontrol-origin: margin;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Convert">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
border-radius:30px;
color: rgb(255, 255, 0);
border-style:outset;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Convert!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Pick a Image Format </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Quit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
border-radius:30px;
color: rgb(255, 255, 0);
border-style:outset;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
